hipo_drf_exceptions/handlers.py

```diff
@@ -17,8 +17,22 @@ def get_fallback_message(exception):
                 # Return first non-empty value in the list. https://github.com/Hipo/hipo-drf-exceptions/issues/8
                 return get_fallback_message(item)
     elif isinstance(exception, dict):
+        # Get message from the first key.
         first_key = next(iter(exception))
-        return get_fallback_message(exception[first_key])
+        message = exception[first_key]
```

Comment:
`message` can list or dict. I'm not sure this will work with them. Did you test with these cases?

Reply:
I tested the case when the `message` is a list (and it works), but could not test for a case when it would be a dictionary. Can you please provide an example when it would be a dictionary?